### DISCUSSION TITLE
Setting session-token cookie to expire with browser session

### DISCUSSION THREAD

**Discussion by saurabhg138** [nextauthjs/next-auth#10277] [Help]
I am using next-auth 4.24.5 version
In my application I want to expire the session when user close out of browser. can this be achieved by next auth.

For now cookie maxAge is not provided by me so it uses  default of 30 days. I tried setting it to undefined so as to make it session cookie 
session: {
    strategy: 'jwt',
    maxAge: undefined,
  },
  but this gives 500 internal server error for http://localhost:3000/api/auth/callback/credentials `TypeError: option expires is invalid`
I tried setting maxAge as undefined due to normal javascript if we dont give cookie lifespan it will expire with session

**Comment by robertogiuliani84** [comment]
I have the same need. Have you solved it somehow?
